consul_config.py.ctmpl

```diff
@@ -53,22 +53,28 @@ MB_DATABASE_URI = 'postgresql://musicbrainz_ro@{{.Address}}:{{.Port}}/musicbrain
 MB_DATABASE_URI = "SERVICEDOESNOTEXIST_pgbouncer-slave_pgbouncer-master"
 {{end}}
 
-SQLALCHEMY_TIMESCALE_URI = ""
-TIMESCALE_ADMIN_URI=""
-TIMESCALE_ADMIN_LB_URI=""
 
+# Use a key 'listenstore_up' in consul (python boolean True or False) to decide if we connect to timescale.
+# If we set SQLALCHEMY_TIMESCALE_URI to an empty string, then the @api_listenstore_needed and @web_listenstore_needed
+# decorators will prevent the webserver from accessing the timescale database
+if {{template "KEY" "listenstore_up"}}:
```

Comment:
This results in a python if statement in the rendered file:
```
if True:
    URI = "real_url"
else:
    URI = ""
```

I'm not sure if there's a difference here between doing it like this or using a consul if statement to render only one of them. Any preference?